src/components/ExecuteButton.js

```diff
@@ -118,106 +35,44 @@ const ExecuteButton = ({
         }
     }, [ remediationStatus ]);
 
-    const [ connected, disconnected ] = data.reduce(
-        ([ pass, fail ], e) => (e.connection_status === 'connected' ? [ [ ...pass, e ], fail ] : [ pass, [ ...fail, e ] ])
-        , [ [], [] ]
-    );
-
-    const rows = [ ...connected, ...disconnected ].map(con =>
-        ({ cells: [
-            {
-                title: con.executor_name
-                    ? <Tooltip content={ `${con.executor_name}` }>
-                        <span>{ con.executor_name.length > 25 ? `${ con.executor_name.slice(0, 22)}...` : con.executor_name }</span>
-                    </Tooltip>
-                    : 'Direct connection'
-
-            },
-            con.system_count,
-            isUserEntitled && { title: styledConnectionStatus(con.connection_status) }
-        ]})
-    );
-    const connectedCount = connected.reduce((acc, e) => e.system_count + acc, 0);
-    const systemCount = data.reduce((acc, e) => e.system_count + acc, 0);
+    const buttonWithTooltip = (isDisabled) => {
+        return (
+            isDisabled
+                ? <Tooltip content='Your account must be configured with Cloud Connector to execute playbooks.'
+                    position='auto'>
+                    <Button
+                        isAriaDisabled={ isDisabled }
```

Comment:
This should just be `isAriaDisabled={true}` or just `isAriaDisabled` since the `isDisabled` check above does a conditional render.